### PR TITLE
eyre: add verbosity levels

### PR DESCRIPTION
**Pull request by tadad**
addresses #6303 , adds a verbosity toggle to %eyre with the `%spew` task, and some new printouts to help with debugging. Verbosity has three levels:
`0` : These will always print out. They are legacy printouts from before this PR that would always fire
`1` : includes discarding channels; subscriptions; cancelling subscriptions; cancelling requests; response errors; authentication errors
`2` : logs every http message response - %start, %cancelling, %completed. It is very verbose, sometimes useful.

not partial to these rankings, open to changing the value on all the messages

### CODE REVIEW

File: pkg/arvo/sys/vane/eyre.hoon [hunk 1]
```diff
@@ -2526,10 +2547,10 @@
       ?>  ?=([%behn %wake *] sign)
       ?^  error.sign
         [[duct %slip %d %flog %crud %wake u.error.sign]~ http-server-gate]
-      =/  discard-channel
-        discard-channel:by-channel:(per-server-event event-args)
+      =*  id  i.t.t.wire 
+      ~>  %slog.[0 leaf+"eyre: {(trip id)} cancelling channel due to timeout"]
```
Comment (belisarius222) [inline pkg/arvo/sys/vane/eyre.hoon:2551]: This looks like a new print.  I think it should use +trace also.

**Review by belisarius222 — CHANGES_REQUESTED** [review]
A couple minor nits, but this is very close.

**Review by belisarius222 — APPROVED** [review]

**Review by belisarius222 — CHANGES_REQUESTED** [review]
Once CI is fixed, I will approve again.

**Comment by tadad** [comment]
Apparently there was another state change between when I started this branch and now. `+load` is working now

**Comment by belisarius222** [comment]
Tests failed, but it looks spurious -- died after 2 minutes in something other than Arvo.  Rerunning.

**Review by belisarius222 — APPROVED** [review]

**Pull request merged**